src/commands/new.js

```diff
@@ -47,6 +47,12 @@ async function command (context) {
     process.exit(exitCodes.PROJECT_NAME)
   }
 
+  // Guard against `ignite new ignite`
+  if (toLower(projectName) === 'ignite') {
+    context.print.error('Crossing the streams is bad! Please use another name for your project.')
```

Comment:
Not super keen on this wording, @fvonhoven, it might be a little confusing. Any other ideas?